projects/qemu/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Copyright 2020 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM gcr.io/oss-fuzz-base/base-builder
+MAINTAINER alxndr@bu.edu
+RUN apt-get update && apt-get install -y make autoconf automake libtool libglib2.0-dev libfdt-dev libpixman-1-dev zlib1g-dev patchelf
+RUN git clone --depth 1 https://git.qemu.org/git/qemu.git qemu
+WORKDIR qemu
+COPY build.sh $SRC/
```

Comment:
Can we store build.sh in the QEMU repository? Maybe as `scripts/oss-fuzz/build.sh`.

Reply:
Based on #3932, this seems possible. I'll send a patch, since its probably a good idea to have this is the repo, regardless.

Reply:
it is recommended to move in your repo - https://google.github.io/oss-fuzz/advanced-topics/ideal-integration/ . Dockerfile has to stay in oss-fuzz repo.

Reply:
`Dockerfile` as it is OK then. Having the build.sh in QEMU repo will ease updating it without opening `oss-fuzz` pull requests. Thanks Abhishek.

Reply:
The patch adding build.sh to QEMU has landed upstream